keyboards/install.php

```diff
@@ -227,12 +227,12 @@ protected static function WriteLinuxBoxes() {
                 <span>Install keyboard</span></a>
               </li>
             </ol>
-          </div>
```

Comment:
Oops. This extra `</div>` broke the remaining look and feel for the install page. Been this way for a little while now...